packages/Pagination/src/Pagination.tsx

```diff
@@ -30,7 +42,9 @@ const Pagination: React.FC<PaginationProps> = ({ options, totalItems, currentPag
     `pagination__item ${number === page ? "pagination__item--active" : ""
     }`;
 
-  const pages = calculatePages();
+  if (totalItems === 0) return null;
+
+  const pages = simple || totalItems === 0 ? null : calculatePages();
```

Comment:
Should totalItems check be `!totalItems`? So we can then not pass in total and it will default to simple?